dashboard/app/controllers/reference_guides_controller.rb

```diff
@@ -22,4 +27,13 @@ def find_reference_guide
       render :not_found
     end
   end
+
+  def find_reference_guides
+    course_version_id = find_matching_course_version(params[:course_course_name])&.id
```

Comment:
unsure again about cancan-ing correctly here. I believe cancan is preventing the course_version from being found at this level, so we can only render :not_found and not :forbidden if the user can't access the unit group.